apps/governance/src/routes/staking/node/nodes-container.tsx

```diff
@@ -38,6 +40,20 @@ export const NodeContainer = ({
         : undefined,
     },
   });
+
+  const [isRefetching, setIsRefetching] = useState(false);
+
+  useEffect(() => {
```

Comment:
This isn't the right approach. If the query errors due to party not found... I'd consider that not an error for the user and just ignore it.

```
if (error && !error.message.includes('failed to get party fo ID')) {
  // render error ui
}
```

You may want an additional hook or some or a utility function if we commonly want to ignore this error. We might also want to not report this error to Sentry.

Reply:
@mattrussell36 I think it might be the right approach. If the API errors, we never get a data object. We need to retry

Reply:
For posterity: the issue was Apollo's default policy for not returning data if there's an error. Setting `errorPolicy: 'all'` in the staking query allowed data to be passed to the children and the error to be ignored